pkg/shoes/shoes.go

```diff
@@ -88,10 +88,10 @@ func (c *GRPCClient) AddInstance(ctx context.Context, runnerName, setupScript st
 	}
 	resp, err := c.client.AddInstance(ctx, req)
 	if err != nil {
-		return "", "", "", fmt.Errorf("failed to AddInstance: %w", err)
+		return "", "", "", datastore.ResourceType(0), fmt.Errorf("failed to AddInstance: %w", err)
 	}
 
-	return resp.CloudId, resp.IpAddress, resp.ShoesType, nil
+	return resp.CloudId, resp.IpAddress, resp.ShoesType, datastore.ResourceType(resp.ResourceType), nil
```

Comment:
Please use unmarshal command https://github.com/whywaita/myshoes/blob/8d9faf2330a46727c6b43ddd6274424f537edd6c/pkg/datastore/resource_type.go#L84

Reply:
Fixed in https://github.com/whywaita/myshoes/pull/169/commits/5a641811fe6c4bd8e4c9ac8b62dc3991bb279e2d